hyperjumptech/monika
# Add city isp info

# Monika Pull Request (PR) 
This PR resolves #365 

## What feature/issue does this PR add  
1. Add city and ISP info to startup message on stdout and notifications

## How did you implement / how did you fix it  
1.  Fetch info from `http://ip-api.com/json/{public_ip}`

## How to test  
1. `npm run start -- -r 1`
2. Should see the information on stdout
3. Should see the information on configured notification channels
![image](https://user-images.githubusercontent.com/9563873/131949425-51056534-1a27-455b-a397-0377fd577340.png)
